src/app/routes/features/features.component.ts

```diff
@@ -15,21 +15,55 @@ import {FeatureRendererComponent} from './feature-renderer.component';
 export class FeaturesComponent implements OnInit {
 
   features: Feature[];
+  filter: string;
   gridApi: GridApi;
   columnApi: ColumnApi;
   gridOptions: GridOptions;
 
   constructor(private featureService: FeatureService, private logger: NGXLogger) {
+      const colDefs: ColDef[] = [
+          {
+              field: 'uid',
+              getQuickFilterText: (params) => {
+                  return params.value;
+              }
```

Comment:
The following block of code can be extracted to a function and re-used

```
(params) => {
                  return params.value;
              }
```

Reply:
How about making this from 
```
getQuickFilterText:  (params) => {
    return params.value;
}
```
to

`getQuickFilterText: (params) => params.value`

as we may not use `params.value` for all the filters